pkgs/development/python-modules/process-tests/default.nix

```
@@ -15,6 +16,14 @@ buildPythonPackage rec {
     hash = "sha256-5dV96nFhJR6RytuEvz7MhSdfsSH9R45Xn4AHd7HUJL0=";
   };
 
+  patches = [
+    (fetchpatch {
+      # Add optional ignore_case param to wait_for_strings
+      url = "https://github.com/ionelmc/python-process-tests/pull/7.diff";
```

```suggestion
      url = "https://github.com/ionelmc/python-process-tests/commit/236c3e83722a36eddb4abb111a2fcceb49cc9ab7.patch";
```
`fetchpatch` produces a fixed-output-derivation, which means that its result should never change. To ensure this, please use the link to the specific commit. The diff of PR # 7 can change, the specific commit can only vanish.